environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - astropy~=4.2
   - pip
   - ctapipe~=0.11
-  - gammapy>=0.18
+  - gammapy=0.18
```

Comment:
Shouldn't this be as in setup.py:
`'gammapy~=0.18.2'`
?

Reply:
No, conda version work differently. This is essentially the same as `==0.18.*` in setup.py terms. It will install the latest 0.18 version, which is 0.18.2 which also won't change since gammapy does not do bugfix releases for older versions.